blocks.js

```diff
@@ -2518,6 +2547,14 @@ BlockMorph.prototype.userMenu = function () {
         'userDestroy'
     );
     menu.addItem(
+      'stringify...',
```

Comment:
The name "scratchblocks" is really unhelpful; to someone unfamiliar with the term, it doesn't really help explain what it is. Maybe "export text" or something?  The closest analogy I can think of is the export project summary feature from Scratch 1.x...